.github/workflows/codegen.yml

```diff
@@ -38,8 +38,20 @@ jobs:
         with:
           go-version: "1.22"
 
+      - name: post-process checkout - copy to github.com
```

Comment:
looks like it is pre-processed.

Reply:
@parth-gr  wrote:
> looks like it is pre-processed.

Actually, it **is** post-processing:

* first we do the checkout-action
* then we do the setup-go action
* And  finally,   we copy the checkout dir into GOPATH.

so the checkout is post-processed after golang has been set up as well.

Reply:
the checkout  **is** actually post-processed as this  does something with the result of thecheckout action.